.github/workflows/cli.yml

```diff
@@ -40,8 +40,7 @@ jobs:
           filters: |
             build:
               - 'cli/**/*'
-              - '.github/**/*'
-              - '!.github/workflows/packages.yml'
+              - '.github/workflows/cli.yml'
```

Comment:
I updated the filter due to it affects all files except `packages.yml`
Do we need to run CLI workflow for anything else?

Reply:
`cli.yml` definitely pulls in a bunch of other files inside `.github/workflows` & `.github/actions`.  We probably want to list them here as well?  or add some more ignores for whatever prompted this PR in the first place?

Reply:
@obmarg wanted to be explicit here 

Looking into cli related workflows 
- `.github/workflows/cli.yml` - included in filter 
- `.github/workflows/cli-partial-release.yml` - triggered by `cli.yml`
- `.github/workflows/sanitize-alert-schedule.yml` - triggered by cron or manually 

So I do not think workflows besides cli should be included. Actions on the other hand should - I will add them

Wdyt?

Reply:
Yeah, we can try that.

Reply:
Added `.github/actions` 👍